aws_cloudwatch_policies/aws_cloudwatch_sensitive_loggroup_encrypted.py

```diff
@@ -0,0 +1,13 @@
+from fnmatch import fnmatch
```

Comment:
I'm kind of a stickler for sorting. Could we name this `aws_cloudwatch_loggroup_sensitive_encrypted.py`?